cityscapesscripts/evaluation/evalPanopticSemanticLabeling.py

```diff
@@ -31,19 +31,7 @@
 from collections import defaultdict
 
 # Image processing
-# Check if PIL is actually Pillow as expected
-try:
-    from PIL import PILLOW_VERSION
-except ImportError:
-    print("Please install the module 'Pillow' for image processing, e.g.")
-    print("pip install pillow")
-    sys.exit(-1)
-
-try:
-    import PIL.Image as Image
-except ImportError:
-    print("Failed to import the image processing packages.")
-    sys.exit(-1)
+import PIL.Image as Image
```

Comment:
```suggestion
from PIL import Image
```

This is typical form as recommended in the docs:

* https://pillow.readthedocs.io/en/stable/handbook/tutorial.html
* https://pillow.readthedocs.io/en/stable/installation.html#installation